### PR TITLE
Release/v0.10.20

### PR DESCRIPTION
**Pull request by Mudiwa66**
CAPI changes

### CODE REVIEW

File: eventstore/management/commands/tests/test_delete_historical_records.py [hunk 1]
```diff
@@ -33,22 +33,23 @@ def test_invalid_arguments(self):
...
-            running_month = running_month + relativedelta(months=1)
+            running_month = running_month + relativedelta(days=31)
 
         self.call_command("Event", 6)
 
         self.assertEqual(Event.objects.count(), 6)
 
     def test_delete_messages(self):
-        running_month = timezone.now() - relativedelta(months=12, hour=12)
+        running_month = timezone.now() - relativedelta(days=365, hour=12)
```
Comment (Mudiwa66) [inline eventstore/management/commands/tests/test_delete_historical_records.py:47]: @erikh360 I'm not sure how to handle leap years or months ending on the 29/30th

Comment (erikh360) [reply eventstore/management/commands/tests/test_delete_historical_records.py:47]: Should be fine for now

File: eventstore/management/commands/tests/test_delete_historical_records.py [hunk 2]
```diff
@@ -33,22 +33,23 @@ def test_invalid_arguments(self):
...
 
         for i in range(12):
             self.create_record(Message, i, running_month)
-            running_month = running_month + relativedelta(months=1)
+            running_month = running_month + relativedelta(days=31)
+            print(f" running month: {running_month}")
```
Comment (erikh360) [inline eventstore/management/commands/tests/test_delete_historical_records.py]: Remove the print please

**Review by erikh360 — COMMENTED** [review]
Just the one comment

File: eventstore/management/commands/tests/test_delete_historical_records.py (hunk 2 above)
Comment (Mudiwa66) [reply eventstore/management/commands/tests/test_delete_historical_records.py]: Removed the print, I'm going to merge now.

**Pull request merged**